### ISSUE TITLE
unable to start Tekno mod - Process crashed on init (Driver conflict)

### ISSUE THREAD

**Issue by imeaniguess** [PkmnYellow/RSV2-Modpack#1]
![image](https://user-images.githubusercontent.com/34157220/233779383-1916ffaf-48de-4070-aa88-254b91ab4c3f.png)

I get this error every time I try to open the bat file or exe

late edit: the Reshade mod is messing up the exe/bat for some reason and I can't run both at once which is weird because I'm assuming they "should" work since they're both in this pack together

**Temporary workaround**: I just turned my computer off and back on in safe mode and everything works 100% fine. I don't know what that means exactly but something I have running etc is interfering with it somehow.

**Comment by PkmnYellow** [comment]
Hmmm, I've never actually encountered that error before with Tekno and Reshade together. :/ And yes, these two should work fine, since

**Comment by PkmnYellow** [comment]
Hmmm, I've never actually encountered that error before with Tekno and Reshade together. :/ And yes, these two should work fine, since it's just a shader injector.

Assuming you're running it in Admin mode, were you able to run Reshade without Tekno? I know integrated GPUs will sometimes, if not always, guarantee a crash at start, from my findings at least.

**Comment by imeaniguess** [comment]
> Were you able to run Reshade without Tekno? I know integrated GPUs will sometimes, if not always, guarantee a crash at start, from my findings at least.

Yeah, I've actually been using it for like 3-4 years before I ever found these other mods. I did a fresh install of everything and I can run it fine but then it just broke my entire game when i tried to add other things. I can't use the Terrorist Hunt Control either. It just does nothing.

**Comment by PkmnYellow** [comment]
Hmmm, do you have the standalone version of Vegas 2 or the Steam version? I can imagine a previous update on the standalone version could cause it to crash.

If you right-click the 'Run TeknoR6Vegas2 (Admin only)' shortcut, and 'Edit', what do you see?

**Comment by imeaniguess** [comment]
> Hmmm, do you have the standalone version of Vegas 2 or the Steam version? 

I have the Steam version
 
> If you right-click the 'Run TeknoR6Vegas2 (Admin only)' shortcut, and 'Edit', what do you see?

![image](https://user-images.githubusercontent.com/34157220/233791534-4b003438-9dc1-4232-be9e-35c5db03b9ea.png)



edit: every time I try to run the bat it opens another process that uses 100% of my CPU and doesn't close
![image](https://user-images.githubusercontent.com/34157220/233792290-4af19ed1-96f8-488e-a36f-8b08c5c99197.png)

**Comment by PkmnYellow** [comment]
The code seems to be in order.

Since it is the Steam version, and you fresh installed, you probably don't have the NOCD exe, and that's mandatory to run Tekno.

You can get that here: https://www.dropbox.com/s/galg0fzrkmvjq16/R6Vegas2_Game.zip?dl=0

Just backup your original game exe, and replace R6Vegas2_Game.exe. Let me know if it works! 

> edit: every time I try to run the bat it opens another process that uses 100% of my CPU and doesn't close

You should probably close those other instances via Task Manager. They probably duplicated from the crashes.

**Comment by imeaniguess** [comment]
Yeah, I've been using that one. I'll try with the one you linked I guess. I just copied [this video](https://youtu.be/_-kCNLP50g4) and did what he did. I've also tried other things like installing the Tekno mod from another site and dragging the files into the Binaries folder (which worked one time, then broke instantly)

**Comment by imeaniguess** [comment]
oops

**Comment by PkmnYellow** [comment]
> Yeah, I've been using that one. I'll try with the one you linked I guess. I just copied [this video](https://youtu.be/_-kCNLP50g4) and did what he did.

Ahhh, I see. Well, knowing that it was duplicating earlier, you should probably make sure no other instance is running before you attempt to run Tekno again.

> oops, didn't mean to hit that, this is my first time every writing messages on here
> 
> so I was able to get the Terrorist Hunt Control and FOV mods working fine at least, I'll see what else I can do

Haha, yeah... no worries. 

Gotcha. Let me know if you discover something.

**Comment by imeaniguess** [comment]
> Gotcha. Let me know if you discover something.

I just turned my computer off and back on in safe mode and everything works 100% fine. I don't know what that means exactly but something I have running etc is interfering with it somehow.

**Comment by PkmnYellow** [comment]
3rd party antiviruses can be a hassle, especially if it's strict, and detecting the .bat's and .exe's as false positives.

**Comment by imeaniguess** [comment]
> 3rd party antiviruses can be a hassle, especially if it's strict, and detecting the .bat's and the .exe as false positives. Do you happen to have Avast, AVG, etc?

Funny, that used to happen to me 10~ years ago when I used AVG and I know it's a major issue with that one in particular but no, I only use Windows Defender and even turned it off which didn't help. I have no idea what's causing the problem but it's something running in the background. Do you think there's an incompatibility issue with my GTX 3090 TI?

**Comment by PkmnYellow** [comment]
> I have no idea what's causing the problem but it's something running in the background.

Unless Windows 10/11 did something unusual with one of their updates, it's probably something you have on startup, maybe?

> Do you think there's an incompatibility issue with my GTX 3090 TI?

Not likely. I know someone from the Vegas 2 discord that has an RTX 3060, and he runs the game with no issues.

**Comment by imeaniguess** [comment]
>Unless Windows 10/11 did something unusual with one of their updates, it's probably something you have on startup, maybe?

After starting my PC normally I turned off and disabled everything possible and it still didn't work so I'm assuming it has something to do with Nvidia since I can't turn that off but in safe mode the drivers are disabled.

Well, if I spam like 100 of them usually one will work. Sometimes a bit more but I still have no idea why this is happening. Safe mode = works fine, still.

Something I've now noticed is that certain attachment options for every weapon type is gone (high capacity magazine, recoil control stock). What mod does that? I only have Teknor, Reshade, Terrorist Control, FOV. I don't want any weapon changes if possible.

**Comment by PkmnYellow** [comment]
> Something I've now noticed is that certain attachment options for every weapon type is gone (high capacity magazine, recoil control stock). What mod does that? I only have Teknor, Reshade, Terrorist Control, FOV. I don't want any weapon changes if possible.

Through installedmods.txt (if you used the batch scripts), it shows what you have currently installed. If you didn't install Vegas 2 Realism or The Complete Overhaul mod, then you should have vanilla weapons.

**Comment by imeaniguess** [comment]
I have no idea what's going on with the drivers or whatever may be hindering my experience and I nearly bricked my PC messing with safe mode so I'm giving up for now haha.

For some reason the ini files were still modified from maybe way earlier when I did have The Complete Overhaul installed so I just put the original files back and it's all good.

**Comment by PkmnYellow** [comment]
> I have no idea what's going on with the drivers or whatever may be hindering my experience and I nearly bricked my PC messing with safe mode so I'm giving up for now haha.

xD I see. Well, I'll keep this issue open if you ever find anything. Sorry I couldn't be of further assistance. 

> For some reason the ini files were still modified from maybe way earlier when I did have The Complete Overhaul installed so I just put the original files back and it's all good.

If you did use the batch scripts, I automatically make backups to revert back to vanilla. Just 'Uninstall' via Run.bat, and you should be good. Or... if you want to do it manually, go to the 'Backup' folder in the game directory and everything should be there to move.
